LKExp-Firmware_update-Serial_update/lib/mbed-os.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-os/
```

Comment:
On peut supprimer ce fichier sauf si tu utiles mbed deploy mais je trouve que le git clone ---depth est plus simple et prend moins de place.

Reply:
je vois qu'il a été déplacé dans APPLICATION

Reply:
Fix in commit f0bb12e
Il est présent dans le [_template](https://github.com/leka/LekaOS_Explorations/tree/master/_template/lib), il faudrait l'en retirer.